cost-analyzer/templates/_helpers.tpl

```diff
@@ -1040,8 +1040,7 @@ Begin Kubecost 2.0 templates
     - name: federated-storage-config
       mountPath: /var/configs/etl/federated
       readOnly: true
-  {{- end }}
-  {{- if .Values.kubecostModel.etlBucketConfigSecret }}
+  {{- else }}
```

Comment:
I am not sure about this. when running as a container, `cloud-cost` doesn't depend on a secret

Reply:
maybe an else if